data/libs/Ship.lua

```diff
@@ -885,12 +885,15 @@ local onGameStart = function ()
 	if loaded_data then
 		CrewRoster = loaded_data
 		Event.Queue('crewAvailable') -- Signal any scripts that depend on initialised crew
-	else
-		CrewRoster = {}
 	end
+
 	loaded_data = nil
 end
 
+local onGameEnd = function()
+	CrewRoster = {}
```

Comment:
This can now be randomly cleared before auto-save on game exit.

Reply:
I thought the pre-exit autosave was emitted via a separate event queued before `onGameEnd` was added to the event queue now?